docs/pages/components/youtube.stories.mdx

```diff
@@ -63,3 +71,21 @@ Then use it like this.
 ```jsx
 <YouTube youTubeId="WUgvvPRH7Oc" />
 ```
+
+To embed a YouTube playlist using the `<YouTube />` component grab the `youTubeListId` from the URL.
+
+```
+https://www.youtube.com/watch?v=C8M-JhhYaBs&list=PL5SvzogSTpeH1Szqw4tPi9ZfgXDbY8GU-
+```
+
+The `youTubeListId` is this bit.
```

Comment:
Does this new functionality refer to playlists? Wondering if the prop should be called `youTubePlaylistId`

Reply:
Yeah, would be more descriptive, right